src/components/message.vue

```diff
@@ -0,0 +1,517 @@
+<template lang=''>
+            <div class='comments_item' :key='comment.postId'>
+                <div class='comment_user_info'>
+                    <img :src='require(`../photos/photosForProfile/${commentAuthor.pathToPhoto}`)' :alt='commentAuthor.accountName' width='32' height='32' />
+                    <p class='comment_user_name'>{{ commentAuthor.accountName }} </p>
+                    <span  v-if='isCurrentUserComment'   class='its_you'>you</span>
+                    <p class='comment_user_date'>{{ dateDifference(comment.date) }}</p>
+                </div>
+              
+                <textarea v-if='isEditable' :value='textareaValue' @input='event => textareaValue = event.target.value' class='comment_user_text comment_user_edit_textarea' />
+       <IsToAddressTheUser v-else :commentText='textareaValue'/>
+                
+                <div class='quantity_of_likes_container'>
+                    <button type='button' class='quantity_calc_button' @click='() => { changeQuantityLikesHandler(Number(comment.quantityOflikes) + 1) }'><img width='10' height='10' src='../photos/+.svg' alt='minus' /></button>
```

Comment:
```suggestion
                    <button type='button' class='quantity_calc_button' @click=changeQuantityLikesHandler(comment.quantityOflikes + 1)'><img width='10' height='10' src='../photos/+.svg' alt='minus' /></button>
```
1. no need to wrap your handlers in arrow functions
2. why do you need to convert `comment.quantityOflikes` to number? just use correct data type in your objects. it should be number, not anything else